ccd-config-generator/src/test/java/uk/gov/hmcts/ccd/sdk/CallbackControllerTest.java

```diff
@@ -155,4 +191,15 @@ CallbackRequest buildRequest(String caseType, String eventId, Map<String, Object
             .build())
         .build();
   }
+
+  private Map<String, Object> caseData() {
```

Comment:
rather than load the json into a map only to reserialise it better to have an overload of makeRequest that accepts the data as a string

Reply:
Discussed with Alex and we decided to leave it as is